src/test/java/org/folio/linked/data/integration/kafka/listener/SourceRecordDomainEventListenerIT.java

```diff
@@ -43,31 +40,39 @@ static void setup(@Autowired KafkaAdminService kafkaAdminService) {
   }
 
   @Test
-  void shouldConsumeSrsDomainEvent() {
+  void shouldNotHandleSrsDomainEvent_whenSourceRecordType_isMarcBib() {
```

Comment:
Let's not delete tests, we will switch this feature on later and we know it for sure. Let's just disable it.

Reply:
done